QNH.Overheid.KernRegister.MailingTool/App.config

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
+  </startup>
+  <appSettings>
+    <add key="host" value="kmail.b3partners.nl"/>
+    <add key="userName" value="kvkmutaties_noordbrabant"/>
+    <add key="password" value="007dd5Aw9"/>
+    <add key="port" value="995"/>
```

Comment:
Ik zie dit nu pas... weet niet of dit daadwerkelijk bruikbare username/password is, maar deze waarden zouden sowieso niet gevuld moeten zijn op github.

@mvdstruijk 
@CorneHogerheijde 
@Sjoske